changelog.md

```diff
@@ -1,3 +1,12 @@
+# 3.11.0
```

Comment:
If you use shrinkwrap the peer dependencies have to be ok. Otherwise you can't build your shrinkwrap file. That's why I consider it as a minor bump.